inc/saimacsec.h

```diff
@@ -364,6 +377,15 @@ typedef enum _sai_macsec_port_attr_t
      */
     SAI_MACSEC_PORT_ATTR_SWITCH_SWITCHING_MODE,
 
+    /**
+     * @brief Control the pause frame
+     *
+     * @type sai_macsec_pause_frame_ctrl_t
+     * @flags CREATE_AND_SET
+     * @default SAI_MACSEC_PAUSE_FRAME_CTRL_ACCEPT
+     */
+    SAI_MACSEC_PORT_ATTR_MACSEC_PAUSE_FRAME_CTRL,
```

Comment:
it is not related to macsec at all, it should be normal port attribute.

Reply:
Fixed.